letsencrypt-apache/letsencrypt_apache/configurator.py

```diff
@@ -230,7 +230,7 @@ def deploy_cert(self, domain, cert_path, key_path,
         if not vhost.enabled:
             self.enable_site(vhost)
 
-    def choose_vhost(self, target_name):
+    def choose_vhost(self, target_name, temp=False):
```

Comment:
You should update the docstring below to say what `temp` does with respect to ssl vhosts and caching.